packages/html-reporter/src/reportView.css

```diff
@@ -29,6 +29,24 @@ body {
   width: 100%;
 }
 
+.report-body {
+  border-radius: 6px;
+  margin: 12px 0 24px 0;
+}
+
+.report-title {
+  flex: none;
+  padding: 8px;
+  font-weight: 400;
+  font-size: 32px !important;
+  line-height: 1.25 !important;
```

Comment:
we usually do pixels

Reply:
I agree, though this is the existing styling from `.test-case-title` in `testCaseView.css`. I don't think we should change that here.